src/test/groovy/com/railway/station/domain/StationFacadeTest.groovy

```diff
@@ -35,7 +36,7 @@ class StationFacadeTest extends Specification implements SampleStations {
     }
 
     def "should list stations"() {
-        given: "we have stations in system Gdańsk, Warszawa i Kraków"
+        given: "we have stations in system Gdańsk, Warszawa and Kraków"
              def stationGdansk = facade.addStation(gdansk)
              def stationWarszawa = facade.addStation(warszawa)
              def stationKrakow = facade.addStation(krakow)
```

Comment:
please think about extracting code from then section. There can be a method validating 3 fields which can be reused across 3 objects